<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatrixTrace</class>
 <widget class="QMainWindow" name="MatrixTrace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matrices trace GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout3">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Matrix 1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox1">
          <property name="minimum">
           <number>-2147483647</number>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget1">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>55</number>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Matrices trace comparison GUI application</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Matrix 2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="openExternalLinks">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox2">
          <property name="minimum">
           <number>-2147483647</number>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget2">
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>35</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>55</number>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Довідка</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Запуск</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MatrixTrace</receiver>
   <slot>onClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>607</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MatrixTrace</receiver>
   <slot>spinBox1ValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MatrixTrace</receiver>
   <slot>spinBox2ValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>768</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MatrixTrace</receiver>
   <slot>documentationClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>783</x>
     <y>550</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClick()</slot>
  <slot>spinBox1ValueChanged(int)</slot>
  <slot>spinBox2ValueChanged(int)</slot>
  <slot>documentationClick()</slot>
 </slots>
</ui>
